Gemfile

```diff
@@ -28,10 +30,11 @@ gem "deface"
 # gem "daemons"
 # gem "delayed_job_active_record"
 # gem "whenever"
-# elsif deploying to Heroku
+# elsif deploying to a PaaS like Heroku
 gem "redis"
 gem "sidekiq"
 group :production do
+  gem "aws-sdk-s3", require: false
   gem "fog-aws"
   gem "rack-ssl-enforcer"
   gem "rails_12factor"
```

Comment:
Again, this two blocs must be inversely commented. This application is deployed in a traditional server.

Reply:
I have got them inversely commented, but as some gems were duplicated after uncommenting them, I have decided to remove duplicated entries